_el/exercises.md

```diff
@@ -1,168 +1,95 @@
+
+> ** Πρόβλημα Parsons **
+>
+> Αναδιατάξτε και μορφοποιήστε τις παρακάτω γραμμές κώδικα ώστε να δίνουν το άθροισμα των θετικών τιμών μιας λίστας.
+> (Μπορεί να χρειαστεί να προσθέσετε και το σύμβολο `;` στα κατάλληλα σημεία.)
+>
 >     total = 0
 >     if v > 0
 >     total += v
 >     for v in values
 
-## Tracing {#s:exercises-tracing}
-
-*Tracing execution* is the inverse of a Parsons Problem: given a few
-lines of code, the learner has to trace the order in which those lines
-are executed. This is an essential debugging skill, and is a good way to
-solidify learners' understanding of loops, conditionals, and the
-evaluation order of function and method calls. The easiest way to
-implement it is to have learners write out a sequence of labelled steps.
-Having them choose the correct sequence from a set (i.e., presenting
-this as an MCQ) adds cognitive load without adding value, since they
-have to do all the work of figuring out the correct sequence, then
-search for it in the list of options.
-
-> **Tracing Execution Order**
-> 
-> In what order are the labelled lines in this block of code executed?
-> 
+
+## Ιχνηλάτηση {#s:exercises-tracing}
+
+*Η Ιχνηλάτηση Εκτέλεσης* είναι το αντίστροφο ενός προβλήματος Parsons: δίνονται μερικές γραμμές κώδικα, και ο μαθητής πρέπει να ανιχνεύσει τη σειρά με την οποία οι γραμμές αυτές εκτελούνται. Αυτή είναι μια σημαντική δεξιότητα εντοπισμού σφαλμάτων και είναι ένας καλός τρόπος να ενισχυθεί η κατανόηση των μαθητών σχετικά με τους βρόχους, τους ελέγχους και τη σειρά κλήσεων μεθόδων. Ο ευκολότερος τρόπος εφαρμογής είναι να ζητηθεί από τους μαθητές να γράψουν τη σειρά εκτέλεσης αριθμημένων βημάτων. Το να ζητηθεί από τους μαθητές να επιλέξουν τη σωστή σειρά από ένα σύνολο σειρών (δηλαδή υπό τη μορφή άσκησης ΕΠΕ), προσθέτει γνωστικό φορτίο χωρίς να υπάρχει κάποια επιπρόσθετη μαθησιακή αξία, καθώς οι μαθητές θα πρέπει αφενός να βρουν τη σωστή σειρά και αφετέρου να την αναζητήσουν σε μια λίστα επιλογών.
```

Comment:
*Η Ιχνηλάτηση Εκτέλεσης*--> το Η έξω από τα αστεράκια

Reply:
Thanks @pdelias ! Fixed in a new commit.